### ISSUE TITLE
Navbar dropdown menu creates page offset when short content is provided

### ISSUE THREAD

**Issue by zapotocnylubos** [DesignRevision/shards-dashboard-pro#3]
## Expected Behavior
Navbar dropdown menu behavior should not be dependant on dropdown toggler size.
If short name provided, dropdown menu width should be expanded inside page (to the left), not outside (to the right).

## Current Behavior
When short name is provided in dropdown toggler in navbar, dropdown menu creates page offset/gap to the right.
[screenshot on imgur.com](https://imgur.com/a/hdkRLBg)

## Steps to Reproduce
- go to [demo page](https://designrevision.com/demo/shards-dashboards/)
- in inspection (F12) (chrome) change user name from **"Sierra Brooks" to "Sierra"** or anything shorter
- open dropdown menu (click it)
- try to scroll horizontally (to the right), you should see gap/offset same as above mentioned screenshot

## Context (Environment)
If you use short user name, for example "admin", it will break whole webpage.

### My personal fix

```
/* Medium devices (tablets, 768px and up) */
@media (min-width: 768px) {
    .dropdown-menu {
        left: auto;
    }
}
```
if works well on all window sizes and all content

**Comment by nickvasdev** [comment]
Great find! 😄The fix for this issue will also be released in the next update! Thank you so much for taking the time to submit these issues!

**Comment by nickvasdev** [comment]
✅This issue has been fixed in 1.3.0.
